apiserver/facades/agent/uniter/newlxdprofile_test.go

```diff
@@ -259,16 +260,20 @@ func (s *newLxdProfileSuite) expectOneLXDProfileRequired() {
 	s.backend.EXPECT().Charm("ch:testme-3").Return(nil, errors.NotFoundf("ch:testme-3"))
 }
 
-func (s *newLxdProfileSuite) expectModel() {
-	s.backend.EXPECT().Model().Return(s.model, nil)
-}
-
 func (s *newLxdProfileSuite) expectModelTypeIAAS() {
-	s.model.EXPECT().Type().Return(state.ModelTypeIAAS)
+	s.modelInfoService.EXPECT().GetModelInfo(gomock.Any()).Return(
+		model.ReadOnlyModel{
+			Type: model.IAAS,
```

Comment:
Will need to update these to include the cloud type once you swap over the current call structure as per above comment.